contracts/payloads/IGP93/PayloadIGP93.sol

```diff
@@ -383,6 +386,93 @@ contract PayloadIGP93 is PayloadIGPMain {
 
     }
 
+    // @notice Action 7: Update expand percentage and duration for fTokens
+    function action7() internal isActionSkippable(7) {
+        // Update fUSDC
+        {
+            FluidLiquidityAdminStructs.UserSupplyConfig[] memory configs_ = new FluidLiquidityAdminStructs.UserSupplyConfig[](1);
+            configs_[0] = FluidLiquidityAdminStructs.UserSupplyConfig({
+                user: address(0), // Protocol address or do we put Lending Factory = 0x54B91A0D94cb471F37f949c60F7Fa7935b551D03
+                token: 0x9Fb7b4477576Fe5B32be4C1843aFB1e55F251B33, // fToken_fUSDC
+                mode: 1,
+                expandPercent: 35 * 1e2, // 35%
+                expandDuration: 6 hours,
+                baseWithdrawalLimit: 0 // Keep existing limit
+            });
+            LIQUIDITY.updateUserSupplyConfigs(configs_);
+        }
+
+        // Update fUSDT
+        {
+            FluidLiquidityAdminStructs.UserSupplyConfig[] memory configs_ = new FluidLiquidityAdminStructs.UserSupplyConfig[](1);
+            configs_[0] = FluidLiquidityAdminStructs.UserSupplyConfig({
+                user: address(0), // Protocol address or do we put Lending Factory = 0x54B91A0D94cb471F37f949c60F7Fa7935b551D03
```

Comment:
here we put fToken address

`token` below would be USDT 

so we configure the limit for the user fToken fUSDT, which uses the token USDT